lms/models/__init__.py

```diff
@@ -5,7 +5,7 @@
 from lms.models.course_groups_exported_from_h import CourseGroupsExportedFromH
 from lms.models.grading_info import GradingInfo
 from lms.models.group_info import GroupInfo
-from lms.models.grouping import CanvasSection, Course, Grouping
+from lms.models.grouping import CanvasGroup, CanvasSection, Course, Grouping
```

Comment:
The `CanvasGroup` import was just forgotten on master?

Reply:
It was not used anywhere (Canvas groups not yet implemented) I think `CanvasGroups` was prematurely added to master (but it was there to demonstrate `Grouping` usage)